payments/base/payment-manager/src/main/java/cm/aptoide/pt/payment_manager/presentation/PaymentMethodsViewModel.kt

```diff
@@ -15,6 +17,7 @@ import java.io.IOException
 class PaymentMethodsViewModel(
   private val purchaseRequest: PurchaseRequest,
   private val paymentManager: PaymentManager,
+  private val lastPaymentStateRepository: LastPaymentStateRepository,
```

Comment:
We should not use repositories from other modules.
Let's stick with our agreements on the modules architecture:
At most view models and business objects (or use cases).